Documentation/network/egress-gateway.rst

```diff
@@ -406,7 +406,7 @@ from the mediabot pod to leave the cluster with the IP of the Egress Gateway nod
 
 .. code-block:: shell-session
 
-    $ kubectl apply -f egress-nat-policy-egress-gateway.yaml
+    $ kubectl apply -f egress-gateway-policy.yaml
```

Comment:
While we're at it, could we replace the "Example policy" direct yaml with a `.. include` for this file from the examples directory? (Could be a follow-up PR, I don't mind).

Reply:
ack, will follow-up :+1: